socketcan_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(socketcan_bridge)
+
+find_package(catkin REQUIRED COMPONENTS can_msgs roscpp socketcan_interface)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES socketcan_to_topic topic_to_socketcan socketcan_interface roscpp can_msgs
```

Comment:
Please remove `socketcan_interface roscpp can_msgs`, `LIBRARIES` is meant for the library targets of this package.
Other libraries from the dependencies will be added automatically by catkin.